janitor/functions.py

```diff
@@ -2920,9 +2920,13 @@ def df(self) -> pd.DataFrame:
         """Get a table of descriptive information in a DataFrame format."""
         return self._get_data_df()
 
+    def __repr__(self):
+        """ Human-readable representation of the `DataDescription` object. """
```

Comment:
pycodestyle linter requests no space after """.

```suggestion
        """Human-readable representation of the `DataDescription` object."""
```

Reply:
grr